src/app/header/login/login.component.scss

```diff
@@ -0,0 +1,8 @@
+h1 {
+  font-weight: 800;
+  margin-bottom: 6%;
+}
+
+h2 {
+  padding-bottom: 25px;
```

Comment:
Prefer margin for inter-component spacing

Reply:
done

Reply:
done